e2e/helpers.ts

```diff
@@ -249,3 +249,31 @@ export async function APISignUpMentor(mentor: any) {
     }),
   });
 }
+
+export async function APIBan(sender: any, reciever: any) {
+  const admin_access_token = await APIAdminAccessToken();
+  const info = await APIUsers(admin_access_token);
+
+  let sender_info = info.find(
+    (o: any) => o.display_name === sender.displayName,
+  );
+  let reciever_info = info.find(
+    (o: any) => o.display_name === reciever.displayName,
```

Comment:
Don't use any please.

Reply:
Should be a linter rule : )

Reply:
But not so bad as it is on test side. But anyways better to avoid. If you are lazy to import stuff you can do stuff inline in TS as typing is structural not nominal,

o: {displayName: string}